packages/ui/components/form/color-picker/colorpicker.test.tsx

```diff
@@ -48,7 +48,7 @@ describe("Tests for ColorPicker component", () => {
     const colorInput = screen.getByRole("textbox");
     await act(async () => userEvent.clear(colorInput));
     const newColorValue = "#00FF00";
-    await act(async () => userEvent.type(colorInput, newColorValue));
+    await act(async () => await userEvent.type(colorInput, newColorValue));
```

Comment:
this seems like a recurring issue, this test is quite flaky, do we event need act ?

Reply:
I had an issue with this in my previous PR https://github.com/calcom/cal.com/pull/12628 where I updated many more packages, including Vite from 4 to 5 in `atoms`, which I now moved to https://github.com/calcom/cal.com/pull/12681 . I added this fix for completeness as I think this will pop up again. `await` is the correct way to handle this

Reply:
@testing-library doc is quite explicit that act should not be necessary, need to double check why we need to chain await like that